src/components/forms/groups/Create.js

```diff
@@ -3,11 +3,61 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { reduxForm, Field, stopSubmit } from 'redux-form';
 
-import Input from 'components/Input';
+import styles from './styles/index.scss';
 import Button from 'components/Button';
 
 const CREATE_GROUP_FORM = 'createGroup';
 
+const renderInputField = ({ input, label, placeholder, meta: { touched, error } }) => (
+  <div>
+    <div>
+      <label>{label}</label>
+    </div>
+    <input className="form-control" {...input} placeholder={placeholder} />
+    {touched && <div className={`text-danger ${styles.error}`}>{error}</div>}
+  </div>
```

Comment:
> My idea was that input component is the entry point for all kind of form inputs: text, selector, checkbox,... Inside we can split it into smaller components. But label, info, errors are the same for all of it.

I don't like this either, but I understand the "label, info, errors are the same for all of it" argument. Can't we create `Input` , `Select`, etc components and wrap them with a shared behaviour?